Explorer/Assets/DCL/Social/WebSocketRpcTransport.cs

```diff
@@ -89,7 +89,18 @@ async UniTaskVoid ListenAndProcessIncomingDataAsync(CancellationToken ct)
                             break;
                         }
                     }
-                    catch (WebSocketException e) { OnErrorEvent?.Invoke(e); }
+                    catch (OperationCanceledException) { break; }
+                    catch (WebSocketException e)
+                    {
+                        OnErrorEvent?.Invoke(e);
+                        break;
+                    }
+                    catch (Exception e)
+                    {
+                        ReportHub.LogException(e, ReportCategory.FRIENDS);
```

Comment:
It should not be `FRIENDS` category the transport relates to the general social features. If there is no suitable category you can pass one to the constructor of the transport from the consumer